### PR TITLE
dynamic log depot machine

### PR DESCRIPTION
**Pull request by kzvyahin**
{{pytest: cfme/tests/configure/ -k collect_log_depot -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6696bd246a63668bdac708390407368ce2dc48f9:

`cfme/tests/configure/test_log_depot_operation.py`:
- :bangbang: Line 200:5: **E301** *expected 1 blank line, found 0*
- :bangbang: Line 207:1: **E303** *too many blank lines (4)*
- :bangbang: Line 338:101: **E501** *line too long (103 > 100 characters)*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0e330ef9c5a674d1b2acbbfac8945bc81425f2fc:

`cfme/tests/configure/test_log_depot_operation.py`:
- :heavy_exclamation_mark: Line 334:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8c5a0e241906823c9b743016916448a6741379bb:

`cfme/tests/configure/test_log_depot_operation.py`:
- :heavy_exclamation_mark: Line 333:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

File: cfme/tests/configure/test_log_depot_operation.py [hunk 1]
```diff
@@ -90,15 +97,25 @@ def pytest_generate_tests(metafunc):
...
         for fixture_name in fixtures:
             assert fixture_name in metafunc.fixturenames
     except AssertionError:
         return
 
+    # deploy template and receiving ip for depot machine
+    depot_provider_key = data["log_db_depot_template"]["provider_key"]
+    depot_template_name = data["log_db_depot_template"]["template_name"]
+    prov = get_mgmt(depot_provider_key)
+    deploy_template(depot_provider_key,
```
Comment (psav) [inline cfme/tests/configure/test_log_depot_operation.py]: This shouldn't be done in collection - we need this in a fixture

**Comment by dajoRH** [comment]
Lint report for commit fd3ee211fe47ffa7d61449d9137403c7e8b31c7b:

`cfme/tests/configure/test_log_depot_operation.py`:
- :heavy_exclamation_mark: Line 334:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 181605bd27cc8e9679b04d13f4ddb4f4250beb3d:

`cfme/tests/configure/test_log_depot_operation.py`:
- :bangbang: Line 200:101: **E501** *line too long (101 > 100 characters)*
- :heavy_exclamation_mark: Line 329:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 000dc650ce2e81d7a0257d4870c604b4491e5e1a:

`cfme/tests/configure/test_log_depot_operation.py`:
- :bangbang: Line 200:101: **E501** *line too long (101 > 100 characters)*
- :heavy_exclamation_mark: Line 329:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8ef6a0ec96cfcdf61a860227b5371ce1e74c1b97:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by jkrocil** [comment]
LGTM! Merging. Nice work @kzvyahin.

Note:
There is definitely some space for refactoring this thing in the near future though. ;)

**Pull request merged**